cmd/memberagent/main.go

```diff
@@ -184,12 +110,74 @@ func main() {
 	}
 	//+kubebuilder:scaffold:builder
 
-	if err := Start(ctrl.SetupSignalHandler(), &hubConfig, memberConfig, hubOpts, memberOpts); err != nil {
+	if err := Start(ctrl.SetupSignalHandler(), hubConfig, memberConfig, hubOpts, memberOpts); err != nil {
 		klog.ErrorS(err, "problem running controllers")
 		os.Exit(1)
 	}
 }
 
+func buildHubConfig(hubURL string, useCAAuth bool, tlsClientInsecure bool) (*rest.Config, error) {
+	var hubConfig = &rest.Config{
+		Host: hubURL,
+	}
+	if useCAAuth {
+		identityKeyFile := os.Getenv("IDENTITY_KEY")
+		identityCertFile := os.Getenv("IDENTITY_CERT")
```

Comment:
nit: identityKeyFile has a different naming convention than tokenFile*Path*. Perhaps we should rename to be consistent.

Reply:
Or update the error?
```
if identityCertFile == "" {
			err := errors.New("identity cert file ***path*** cannot be empty")
```

Reply:
fixed